client-sdk/ts-web/core/playground/src/index.js

```diff
@@ -1,147 +1,141 @@
 // @ts-check
 
 import * as oasis from './../..';
-import * as shared from './shared';
 
 const nic = new oasis.client.NodeInternal('http://localhost:42280');
 
-(async function () {
-    try {
```

Comment:
this is best viewed with hidden whitespace changes. we're outdenting the whole script a level due to taking out the `try`